background.js

```diff
@@ -171,6 +171,15 @@ ipcMain.on('k8s-restart', async (event) => {
   }
 });
 
+app.on('window-preferences', () => {
```

Comment:
I didn't love tacking random events onto the app; I think a better fit would be to convert the tray stuff to a class that extends `EventEmitter`, and have that emit commands instead.  But that seemed a bit big for that PR — I'd be happy to do a follow-up to implement that (which would be slightly easier to review than doing it in this one).

Reply:
SGTM. It can be here or a follow-up. Your call.

Reply:
I'd prefer in a follow-up, just to keep the PRs small and focused(-ish).